Yubico.YubiKey/src/Yubico/YubiKey/YubiKeyFeature.cs

```diff
@@ -15,251 +15,256 @@
-        /// Configure the slot to allow for user-triggered static password change.
+        ///     Configure the slot to allow for user-triggered static password change.
         /// </summary>
         OtpPasswordManualUpdates,
 
         // PIV application features
 
         /// <summary>
-        /// An attestation statement which is an X.509 certificate that certifies a
-        /// private key was generated by a YubiKey.
+        ///     An attestation statement which is an X.509 certificate that certifies a
+        ///     private key was generated by a YubiKey.
         /// </summary>
         PivAttestation,
 
         /// <summary>
-        /// Ability to use an AES key as the PIV management key. A YubiKey
-        /// that can set the management key to AES, can still set it to
-        /// Triple-DES.
+        ///     Ability to use an AES key as the PIV management key. A YubiKey
+        ///     that can set the management key to AES, can still set it to
+        ///     Triple-DES.
         /// </summary>
         PivAesManagementKey,
 
         /// <summary>
-        /// Ability to get data about the key in a slot.
+        ///     Ability to get data about the key in a slot.
         /// </summary>
         PivMetadata,
 
         /// <summary>
-        /// The cryptographic RSA algorithm with the key size 1024 bits
-        /// supported by the PIV Application on the YubiKey.
+        ///     The cryptographic RSA algorithm with the key size 1024 bits
+        ///     supported by the PIV Application on the YubiKey.
         /// </summary>
         PivRsa1024,
 
         /// <summary>
-        /// The cryptographic RSA algorithm with the key size 2048 bits
-        /// supported by the PIV Application on the YubiKey.
+        ///     The cryptographic RSA algorithm with the key size 2048 bits
+        ///     supported by the PIV Application on the YubiKey.
         /// </summary>
         PivRsa2048,
 
         /// <summary>
-        /// The cryptographic RSA algorithm with the key size 3072 bits
-        /// supported by the PIV Application on the YubiKey.
+        ///     The cryptographic RSA algorithm with the key size 3072 bits
+        ///     supported by the PIV Application on the YubiKey.
         /// </summary>
         PivRsa3072,
 
         /// <summary>
-        /// The cryptographic RSA algorithm with the key size 4096 bits
-        /// supported by the PIV Application on the YubiKey.
+        ///     The cryptographic RSA algorithm with the key size 4096 bits
+        ///     supported by the PIV Application on the YubiKey.
         /// </summary>
         PivRsa4096,
 
         /// <summary>
-        /// The cryptographic ECC algorithm with the parameters P-256,
-        /// specified in FIPS 186-4 supported by the PIV Application on the YubiKey.
+        ///     The cryptographic ECC algorithm with the parameters P-256,
+        ///     specified in FIPS 186-4 supported by the PIV Application on the YubiKey.
```

Comment:
I'm a little confused by this. Do you mean "The cryptographic ECC algorithm with the parameters P-256, as specified in FIPS 186-4. Supported by the PIV Application on the YubiKey."?

Reply:
@AdamVe

Reply:
The parameters for ECC algorithms are specified in FIPS 186-5 document, and the enum value `PivEcc256` refers to that ECC algorithm with the P-256 parameters and (the enum value) is supported by the PIV application on the YubiKey. 

I would personally use here: 

```
///    The cryptographic ECC algorithm with the parameters P-256,
///    specified in FIPS 186-4, supported by the PIV Application on the YubiKey.
```
or (as you suggested)
```
///    The cryptographic ECC algorithm with the parameters P-256,
///    as specified in FIPS 186-4, supported by the PIV Application on the YubiKey.
```
Your version is also correct, so please, choose what it best from your experience.

Note: It looks that for many lines in the pull request _only formatting_ has been updated (see the spaces after the `///`) and not the actual comment content. It is great if also review these and find such issues as this one :)

Reply:
Awesome, thanks!